lib/bolt/node/ssh.rb

```diff
@@ -260,6 +260,7 @@ def _run_command(command)
     end
 
     def _run_script(script, arguments)
+      # This is redundant with the message in node.rb
       @logger.info { "Running script '#{script}'" }
```

Comment:
This should probably happen in the executor not each transport to avoid duplication

Reply:
It currently happens in both `node.rb` and `node/transport.rb`. Simplest solution is to remove the logs in `transport.rb`, should we do that and also move the `node.rb` log to the executor? 
https://github.com/puppetlabs/bolt/blob/master/lib/bolt/node.rb#L84
https://github.com/puppetlabs/bolt/blob/master/lib/bolt/node/ssh.rb#L263

Reply:
the run_ functions in node.rb are going away in BOLT-287 only logging this in the executor is probably best.